### PR TITLE
Feat/star in slices

### PR DESCRIPTION
**Pull request by nehasri89**
This PR adds the stars in all tiles. The star is in same level as flag, If there are more than one flag then it will wrap on the second line to give space to the star

PFA the screenshots

Screenshot 2019-04-08 at 16 33 23
Screenshot 2019-04-08 at 16 34 08
Screenshot 2019-04-08 at 16 35 03

For Secondary Four, the stars stick to bottom right

Screen Shot 2019-04-10 at 17 10 18

### CODE REVIEW

File: packages/edition-slices/src/tiles/shared/styles/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    marginTop: -5,
+    width: "25%"
+  }
+};
+
+const verticalStyles = {
+  container: {
+    flexDirection: "column"
+  },
+  flagStyle: {},
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/shared/styles/index.js]: do we need that ^

File: packages/edition-slices/src/tiles/tile-ac/styles/index.js [hunk 2]
```diff
@@ -14,6 +15,9 @@ const styles = {
   imageContainer: {
     width: "100%"
   },
+  star: {
+    ...verticalStyles
+  },
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-ac/styles/index.js]: do we expect additional styling for every tile or we can do `star: verticalStyles`

File: packages/pages/src/section/section.js [hunk 3]
```diff
@@ -27,6 +27,7 @@ class SectionPage extends Component {
     };
     this.onAppStateChange = this.onAppStateChange.bind(this);
     this.toggleArticleSaveStatus = this.toggleArticleSaveStatus.bind(this);
+    this.syncAppData = this.syncAppData.bind(this);
```
Comment (ValGeorgiev) [inline packages/pages/src/section/section.js]: why we need this binding?

Comment (tuncaulubilge) [reply packages/pages/src/section/section.js]: I slipped this one in, was missing from the previous PR for data sync. `syncAppData` is using props, so we needed to bind it.

File: packages/edition-slices/src/tiles/tile-ac/styles/index.js (hunk 2 above)
Comment (nehasri89) [reply packages/edition-slices/src/tiles/tile-ac/styles/index.js]: we might have diff styles per tile, but for now of you if you prefer I can change it.

File: packages/pages/src/section/section.js (hunk 3 above)
Comment (ValGeorgiev) [reply packages/pages/src/section/section.js]: ahh didn't saw ComponentDidMount, okay

File: packages/edition-slices/src/tiles/shared/tile-summary.js [hunk 4]
```diff
@@ -26,12 +29,30 @@ class TileSummary extends Component {
   renderFlags() {
     const {
       tile: {
-        article: { expirableFlags }
+        article: { expirableFlags, id }
       },
-      flagColour
+      flagColour,
+      isDarkStar,
+      isStar,
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/shared/tile-summary.js]: Can we call this `withStar`?

File: yarn.lock [hunk 5]
```diff
@@ -1861,6 +1861,51 @@
     webpack-dev-server "3.1.7"
     webpack-merge "^4.1.1"
 
+"@times-components/schema@0.4.2":
```
Comment (tuncaulubilge) [inline yarn.lock]: These should go away if you fix the versions, do a `yarn deps:fix` and yarn again

**Pull request merged**